cypress/integration/parallel-3/article.interactivity.spec.js

```diff
@@ -46,23 +46,24 @@ describe('Interactivity', function () {
 				// which case this check should be updated
 				.should('eq', 2);
 		});
+		// eslint-disable-next-line mocha/no-exclusive-tests
 		describe('When most viewed is mocked', function () {
+			// eslint-disable-next-line mocha/no-mocha-arrows
```

Comment:
Ditto here, is this linting exclusion required?

Reply:
Removed